src/si/momentum.rs

```diff
@@ -0,0 +1,201 @@
+
+            test::<m::kilogram, l::yottameter, t::second, mmm::kilogram_yottameter_per_second>();
+            test::<m::kilogram, l::zettameter, t::second, mmm::kilogram_zettameter_per_second>();
+            test::<m::kilogram, l::exameter, t::second, mmm::kilogram_exameter_per_second>();
+            test::<m::kilogram, l::petameter, t::second, mmm::kilogram_petameter_per_second>();
+            test::<m::kilogram, l::terameter, t::second, mmm::kilogram_terameter_per_second>();
+            test::<m::kilogram, l::gigameter, t::second, mmm::kilogram_gigameter_per_second>();
+            test::<m::kilogram, l::megameter, t::second, mmm::kilogram_megameter_per_second>();
+            test::<m::kilogram, l::kilometer, t::second, mmm::kilogram_kilometer_per_second>();
+            test::<m::kilogram, l::hectometer, t::second, mmm::kilogram_hectometer_per_second>();
+            test::<m::kilogram, l::decameter, t::second, mmm::kilogram_decameter_per_second>();
+            test::<m::kilogram, l::decimeter, t::second, mmm::kilogram_decimeter_per_second>();
+            test::<m::kilogram, l::centimeter, t::second, mmm::kilogram_centimeter_per_second>();
+            test::<m::kilogram, l::millimeter, t::second, mmm::kilogram_millimeter_per_second>();
+            test::<m::kilogram, l::micrometer, t::second, mmm::kilogram_micrometer_per_second>();
+            test::<m::kilogram, l::nanometer, t::second, mmm::kilogram_nanometer_per_second>();
+            test::<m::kilogram, l::picometer, t::second, mmm::kilogram_picometer_per_second>();
+            test::<m::kilogram, l::femtometer, t::second, mmm::kilogram_femtometer_per_second>();
+            test::<m::kilogram, l::attometer, t::second, mmm::kilogram_attometer_per_second>();
+            test::<m::kilogram, l::zeptometer, t::second, mmm::kilogram_zeptometer_per_second>();
+            test::<m::kilogram, l::yoctometer, t::second, mmm::kilogram_yoctometer_per_second>();
+
+            test::<m::ton, l::meter, t::second, mmm::ton_meter_per_second>();
+            test::<m::kilogram, l::meter, t::minute, mmm::kilogram_meter_per_minute>();
+            test::<m::kilogram, l::meter, t::hour, mmm::kilogram_meter_per_hour>();
+            test::<m::kilogram, l::meter, t::day, mmm::kilogram_meter_per_day>();
+
+            test::<m::slug, l::foot, t::second, mmm::slug_foot_per_second>();
+            test::<m::slug, l::inch, t::second, mmm::slug_inch_per_second>();
+            test::<m::pound, l::foot, t::second, mmm::pound_mass_foot_per_second>();
+            test::<m::pound, l::inch, t::second, mmm::pound_mass_inch_per_second>();
+
+            fn test<M: m::Conversion<V>, L: l::Conversion<V>, T: t::Conversion<V>, R: mmm::Conversion<V>>() {
+                Test::assert_approx_eq(&Momentum::new::<R>(V::one()),
+                    &(Mass::new::<M>(V::one()) * Length::new::<L>(V::one()) / Time::new::<T>(V::one())));
```

Comment:
Wrap just before the `/` so the line doesn't go over 100 characters. Leave the `fn test` line for the moment. Not sure how that should be formatted.

Reply:
Maybe using `where` clauses (I *think* those are stable on the oldest supported target)?

Reply:
Yeah, go ahead and use a `where` clause. They're used everywhere else. I think I was just trying to save horizontal space in the test functions.

Reply:
sgtm, done